charts/dbildungs-iam/Chart.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: v2
 name: dbildungs-iam
```

Comment:
i think the helm chart name and repo name should align and this should be corrected

Reply:
Planned for later, probably together with moving the keycloak subchart into it's own repository in the next PI topic.